xenon/bot.py

```diff
@@ -64,6 +64,10 @@ def __init__(self, *args, **kwargs):
         if message.author.bot:
             return
 
+        if message.content.startswith(self.config.prefix) and self.config.private_bot == True and self.config.owner_id != message.author.id:
+            await message.channel.send("Only my owner is allowed to use me!")
```

Comment:
I hate to sound picky but this doesn't really follow the "standard" of all of xenon's other outputs. Take a look at some of the other outputs, you'll see what I mean :)
Usually everything is in embeds and it's never really in "first person", but this is up to Merlin.

Reply:
Alright, let me find another error message... 🤔

Reply:
Changed, what do you think?